Add info about count store (#635)
neo4j/docs-cypher: modules/ROOT/pages/functions/aggregating.adoc

|===

> [NOTE]

Reviewer: I think a `TIP` would be more appropriate.

====

> Neo4j maintains a transactional count store for holding count metadata, which significantly increases the speed of queries using the `count()` function.

Reviewer: ```suggestion
Neo4j maintains a transactional count store for holding count metadata, which can significantly increase the speed of queries using the `count()` function.
```

> More information about the count store can be read in this link:https://neo4j.com/developer/kb/fast-counts-using-the-count-store/[Neo4j Knowledge Base article].

Reviewer: ```suggestion
For more information about the count store, refer to link:https://neo4j.com/developer/kb/fast-counts-using-the-count-store/[Fast counts using the count store].
```

I think it's always nicer to use the page title, when it doesn't disrupt the style, so as a user I know what I'm gonna get if I click.
